deploy/use_pre_provisioned_batch_account.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+PRJ_ROOT="$(cd `dirname "${BASH_SOURCE}"`/..; pwd)"
+
+environment_code=${1}
+
+target_batch_account_name=${2}
```

Comment:
We might need to have this not driven by the user because there are multiple deployments sharing a common Batch Account. So, there is a possibility of pool name collision especially using the pool names from README.md. Can you make this follow a naming convention like envCode-data-cpu-pool?

Reply:
that is how it works, pool-name by default is `envCode-data-cpu-pool` which a user can override if they want to